snooty.toml

```diff
@@ -4,13 +4,13 @@ intersphinx = ["https://www.mongodb.com/docs/manual/objects.inv"]
 toc_landing_pages = ["/introduction", "/tutorials", "/sink-connector", "/source-connector", "/sink-connector/configuration-properties", "/source-connector/configuration-properties", "/sink-connector/fundamentals", "/source-connector/fundamentals", "/source-connector/usage-examples", "/security-and-authentication", "/troubleshooting"]
 
 [constants]
-mkc = "MongoDB Kafka Connector"
+connector = "MongoDB Kafka Connector"
 connector-long = "MongoDB Connector for Apache Kafka"
-ak = "Apache Kafka"
-kc = "Kafka Connect"
-avro = "Apache Avro"
-avro-short = "Avro"
-kafka-connect = "Confluent Kafka Connect"
+apache-kafka = "Apache Kafka"
+kafka = "Kafka Connect"
```

Comment:
Issue:
I think that "kafka" should represent Apache Kafka and not "Kafka Connect". Apache Kafka is a standalone product name and "Kafka Connect" is a different product that's ambiguous since there's "Confluent Kafka Connect" and "Apache Kafka Connect".

Suggestion:
- `kafka = "Apache Kafka"`
- Remove `apache-kafka`
- `kafka-connect = "Confluent Kafka Connect"`// I don't think we ever reference the Apache version (Apache Kafka Connect), so it seems safe to use Confluent's product, similar to how `connector = "MongoDB Kafka Connector"` is defined
- Remove `confluent-kafka`